src/Compilers/Extension/CompilerExtension.csproj

```diff
@@ -21,6 +21,7 @@
     <IncludeDebugSymbolsInLocalVSIXDeployment>true</IncludeDebugSymbolsInLocalVSIXDeployment>
     <CopyBuildOutputToOutputDirectory>true</CopyBuildOutputToOutputDirectory>
```

Comment:
As a side note, nothing to do with this change, is this and the pdb supposed to be true?